exercises/pangram/src/test/kotlin/PangramTest.kt

```diff
@@ -12,25 +12,25 @@ class PangramTest {
 
     @Ignore
     @Test
-    fun perfectLowercasePangram() {
+    fun perfectLowerCase() {
```

Comment:
Here and below. Let's use description in backticks for test names.
E.g. [`empty sentence`](https://github.com/exercism/problem-specifications/blob/master/exercises/pangram/canonical-data.json#L10) or [`perfect lower case`](https://github.com/exercism/problem-specifications/blob/master/exercises/pangram/canonical-data.json#L18).

Reply:
Will do it later.

Reply:
Apologies, on holiday at the moment.

Reply:
Don't worry. :)
Enjoy your vacation!